examples/snuba-lw-deletions/1/snuba-lw-deletions-search-issues.json

```diff
@@ -0,0 +1,13 @@
+{
+  "storage_name": "search_issues",
+  "rows_to_delete": 1000,
```

Comment:
Is snuba going to honor the value of `"rows_to_delete"` that is passed in here? I thought producer-consumer will determine the batch size.

Reply:
I believe this field represents "how many rows does this delete effect". (ex. 1000)
The consumer will use this to increment the overall batch size.

Reply:
yeah this attribute is going to be used to batch up messages - though i've been thinking that maybe it should be part of the headers for the message

Reply:
in general, we don't really recommend putting important information in the headers of the message. i think it's more error prone, not covered by the schema, and needs to be manually serialized/deserialized. it also doesn't work with the snuba backend in dev that bypasses kafka